app/src/app.tsx

```diff
@@ -10,14 +10,15 @@ import SubmissionTextBox from "./components/submissionPage/SubmissionTextBox";
 import {
     getCurrentUserName,
     onAuthStateChanged,
-    ProfileData,
-} from "./firebase/AuthService";
+} from "./firebase/users/UserService";
```

Comment:
What's the rationale behind renaming this from `AuthService` to `UserService`?

Reply:
This is because that module not only handles authentication but also user profile related stuff and creating users